dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -120,6 +129,28 @@ async def get_job_info(self):
 
         return jobs
 
+    async def get_job_submission_info(self):
+        """Info for Ray job submission.  Here a job can have 0 or many drivers."""
+
+        jobs = {}
+
+        for job_submission_id, job_data in self._job_data_client.get_all_jobs().items():
+            if job_data is not None:
+                status_info = job_data.status_info
+                entry = {
+                    "status": None if status_info is None else status_info.status,
+                    "status_message": None
+                    if status_info is None
+                    else status_info.message,
+                    "start_time": job_data.start_time,
+                    "end_time": job_data.end_time,
+                    "namespace": job_data.namespace,
+                    "metadata": job_data.metadata,
```

Comment:
In the old job snapshot, the metadata contains the job ID:

```
'metadata': {
    'jobName': 'raysubmit_RcjHs96ykN3SQDB1',
    'jobSubmissionId': 'raysubmit_RcjHs96ykN3SQDB1',
    'rayTest789': '789'
},
```

In the new job snapshot, `metadata` just consists of the `metadata` kwarg passed in `submit_job`:

```
'metadata': {'rayTest789': '789'}
```

Is this what we want?

Reply:
yes that looks right to me! cc @tchordia